developer/src/tike/main/UfrmMain.pas

```diff
@@ -605,7 +605,10 @@ procedure TfrmKeymanDeveloper.FormCreate(Sender: TObject);
   if TikeCommandLine.StartupProjectPath <> '' then
   begin
     try
-      LoadGlobalProjectUI(ptUnknown, TikeCommandLine.StartupProjectPath);
+      if LoadGlobalProjectUI(ptUnknown, TikeCommandLine.StartupProjectPath) = nil then
+      begin
+        raise EProjectLoader.Create('Project is already open in another process');
```

Comment:
Does the lock file get left behind if the user reboots/shuts down while Keyman Developer is open?

I get frustrated by Android Studio when it does this. They include the path of the lock file to remove, so the user can manually remove it.

Reply:
Two things:

1. The lock file is automatically deleted on close. It would take a hard reset for it to remain
2. Keyman Developer will attempt to recreate it even if it exists -- so it does no real damage to leave it there in any case (and it's out of the way of the user now so it's really very harmless).